src/main/java/org/reaktivity/nukleus/http2/internal/routable/stream/SourceInputStreamFactory.java

```diff
@@ -419,9 +460,7 @@ private int decodePreface(final DirectBuffer buffer, final int offset, final int
                                                  .build();
```

Comment:
now Target#doSettings() handles sending SETTINGS frame. This also eliminates allocation of buffers.